eggo/fabric_cli.py

```diff
@@ -124,35 +125,49 @@ def do():
 
     execute(do, hosts=get_worker_hosts())
 
+def sudo(command, **kwargs):
```

Comment:
Is this function necessary?  Couldn't you just swap out the fabric `run` with the fabric `sudo` where necessary?  I'd guess that `sudo cmd` would work normally even if the current user is root, no?